pkg/config/validation/validation.go

```diff
@@ -1939,8 +1939,9 @@ func ValidateControlPlaneAuthPolicy(policy meshconfig.AuthenticationPolicy) erro
 	return fmt.Errorf("unrecognized control plane auth policy %q", policy)
 }
 
-func validateWorkloadSelector(selector *type_beta.WorkloadSelector) error {
+func validateWorkloadSelector(selector *type_beta.WorkloadSelector) Validation {
 	var errs error
```

Comment:
Its preferred to have the style the rest of the package uses.

```
v := Validation{}
v = appendValidation(v, WrapWarning(some warning)))
v = appendValidation(v, fmt.Errorf(some error)))
return v.Unwrap()
```

Reply:
Updated.